baseplate/core.py

```diff
@@ -187,6 +190,7 @@ def configure_tracing(self, service_name,
                 num_span_workers=num_span_workers,
                 span_batch_interval=span_batch_interval,
                 sample_rate=sample_rate,
+                log_if_unconfigured=log_if_unconfigured,
```

Comment:
I just took a quick stab at this. I think reworking this will require refactoring `TraceBaseplateObserver` to take a client, and doing that basically ripples throughout all the existing tracing stuff.